src/main/java/net/snowflake/ingest/streaming/internal/DataValidationUtil.java

```diff
@@ -90,14 +90,6 @@ class DataValidationUtil {
     objectMapper.registerModule(module);
   }
 
-  /**
-   * Creates a new SnowflakeDateTimeFormat. In order to avoid SnowflakeDateTimeFormat's
-   * synchronization blocks, we create a new instance when needed instead of sharing one instance.
-   */
-  private static SnowflakeDateTimeFormat createDateTimeFormatter() {
-    return SnowflakeDateTimeFormat.fromSqlFormat("auto");
```

Comment:
unused?

Reply:
Yeah, this was an unused method from the previous release.